.github/workflows/test_build_deploy.yml

```diff
@@ -323,6 +341,66 @@ jobs:
           name: webapp-dist
           path: ./server/dist
 
+  update_helm_chart:
+    name: Update Helm chart
+    runs-on: ubuntu-latest
+
+    needs: test_build_deploy
+
+    steps:
+      - name: Obtaining release information artifact
+        id: release-info-artifact
+        uses: actions/download-artifact@v2
+        continue-on-error: true
+        with:
+          name: release-info.json
+      - name: Indicating whether release info exist
+        id: release-info-file
+        env:
+          ARTIFACT_LOCAL_PATH: '${{ steps.release-info-artifact.outputs.download-path }}/release-info.json'
+        shell: bash
+        run: |
+          test -s "${ARTIFACT_LOCAL_PATH}" && echo '::set-output name=exists::true'
+          echo "::set-output name=releaseInfo::$(cat ${ARTIFACT_LOCAL_PATH})"
+
+      - name: Checking out 'wire-server'
+        uses: actions/checkout@v2
+        if: ${{ steps.release-info-file.outputs.exists == 'true' }}
+        with:
+          repository: 'wireapp/wire-server'
+          fetch-depth: 1
+
+      - name: Changing Helm value of the webapp chart
+        id: change-helm-value
+        if: ${{ steps.release-info-file.outputs.exists == 'true' }}
+        shell: bash
+        run: |
+          sed --in-place --expression="s/  tag: .*/  tag: \"${{ fromJSON(steps.release-info-file.outputs.releaseInfo).imageTag }}\"/" ./charts/webapp/values.yaml
+          git add ./charts/webapp/values.yaml
+          echo "::set-output name=releaseUrl::${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}/releases/tag/${{ fromJSON(steps.release-info-file.outputs.releaseInfo).releaseName }}"
+
+      - name: Creating Pull Request
+        id: create-pr
+        if: ${{ steps.release-info-file.outputs.exists == 'true' }}
+        uses: peter-evans/create-pull-request@v3
+        with:
+          draft: true
+          token: ${{ secrets.ZEBOT_GH_TOKEN }}
+          author: 'Zebot <zebot@users.noreply.github.com>'
+          branch: charts-update-webapp-image-tag-${{ github.run_number }}
+          commit-message: 'chore: [charts] Update webapp version'
+          title: 'Update webapp version in Helm chart [skip ci]'
+          body: |
+            Image tag: `${{ fromJSON(steps.release-info-file.outputs.releaseInfo).imageTag }}`
+            Release: [`${{ fromJSON(steps.release-info-file.outputs.releaseInfo).releaseName }}`](${{ steps.change-helm-value.outputs.releaseUrl }})
+
+      - name: Printing Pull Request URL
+        if: ${{ steps.release-info-file.outputs.exists == 'true' }}
+        shell: bash
+        run: |
+          echo "PR: ${{ steps.create-pr.outputs.pull-request-url }}"
```

Comment:
```suggestion
        run: echo "PR: ${{ steps.create-pr.outputs.pull-request-url }}"
```

Reply:
Sadly, this wont work because of the colon after `PR:`. At least that is what my editor says. It might be a false positive though. I guess, there is no harm in keeping it that way.